src/main/java/net/atos/zac/app/formulieren/FormulierDefinitieRESTService.java

```diff
@@ -11,14 +11,7 @@
 
 import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
-import jakarta.ws.rs.Consumes;
-import jakarta.ws.rs.DELETE;
-import jakarta.ws.rs.GET;
-import jakarta.ws.rs.POST;
-import jakarta.ws.rs.PUT;
-import jakarta.ws.rs.Path;
-import jakarta.ws.rs.PathParam;
-import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.*;
```

Comment:
STAR IMPORTS should not be used !!

Reply:
Don't agree on not using the star import :-) But I have removed it and added the individual imports.

Reply:
At INFO we have a policy in our projects not to use wildcard imports, mainly for the reason to avoid unforeseen class name conflicts. But yeah, pros and cons as always so mostly a matter of choosing a policy and sticking with it. ;-) https://www.baeldung.com/java-wildcard-imports